sbnobj/Common/Calibration/TrackCaloSkimmerObj.h

```diff
@@ -312,8 +314,9 @@ namespace sbn {
     TrackTruth truth; //!< Truth-matching information
 
     TrackInfo():
-      t0(-1),
-      t0CRT(-1),
+      t0PFP(-1),
+      t0CRTTrack(-1),
+      t0CRTHit(-1),
```

Comment:
Sorry to be really picky! Is it worth choosing an initialisation value that is non-physical? -1 is a bit risky for timing variables, could be physical depending on the unit / reference frame. (e.g. 1us before the event trigger).

I see the limits header is already included, how about this:

```suggestion
      t0PFP(std::numeric_limits<float>::lowest()),
      t0CRTTrack(std::numeric_limits<float>::lowest()),
      t0CRTHit(std::numeric_limits<float>::lowest()),
```